types/storage_change_set.go

```diff
@@ -1,6 +1,7 @@
 // Go Substrate RPC Client (GSRPC) provides APIs and types around Polkadot and any Substrate-based chain RPC calls
 //
 // Copyright 2019 Centrifuge GmbH
+// Copyright 2021 Snowfork
```

Comment:
Should this be in other files we modified too? @musnit

Reply:
sure, or just remove it. per-file copyrights are a bit ridiculous